apps/gamut-mapping/index.html

```diff
@@ -59,7 +59,7 @@ <h2>Gamut mapped</h2>
 				</dd>
 				<dd>
 					<css-color swatch="large" :color="mapped[method].color"></css-color>
-					<dl class="deltas" v-if="!color.inGamut('p3')">
```

Comment:
Why?

Reply:
This was hiding instances where the gamut mapping methods were still changing the color, even if it was in gamut.

Reply:
That is helpful info; a true relative colorimetric GMA should not modify any in-gamut color but in practice, some methods will do so either because of numerical inacuracies/epsilon or deliberately, to achieve a smoother transition at the gamut boundary.

Reply:
I separated this out into #458